models/ac_generator.cpp

```diff
@@ -163,11 +163,9 @@ nest::ac_generator::ac_generator( const ac_generator& n )
  * ---------------------------------------------------------------- */
 
 void
-nest::ac_generator::init_state_( const Node& proto )
+nest::ac_generator::init_state_()
 {
```

Comment:
Should there also be an explicit call (in this and similar cases below) to the superclass init_state_() (here: ``DeviceNode::init_state_()``) as a first line of the function?

Reply:
@clinssen `DeviceNode` does not have an `init_state_()`. Calling `DeviceNode::init_state_()` implicitly calls `Node::init_state_()` (as `Node` is the parent of `DeviceNode`), and this function is empty. `Device::init_state_()` exists, and is also empty, but I guess this is handled with the `device_.init_state_()` call.